### ISSUE TITLE
Start work on "model reconstruction" for two established external models

### ISSUE THREAD

**Issue by alysbrett** [ScottishCovidResponse/SCRCIssueTracking#89]
This is an initial issue to kick off the "model reconstruction" project within SCRC. The idea is to fork two existing models into our Github organisation with a view to being able to run them and understand their inputs and outputs to compare to our own. Initial input from RSE(s) is needed to fork and try running the models and understand how to handle data etc, and from epidemiologist(s) to examine what comparisons would be useful and interpret inputs and outputs.

Here are the initial thoughts from @richardreeve that instigated this work:

At some point soon we're going to be trying to compare our models to each other. However, if we get that up and running, we won't know who's right if they disagree. Meanwhile, first LSHTM and then Imperial have released their models... what I'm thinking is that we could put people to work understanding the code in those models, maybe forking them into our repos to adapt them to use our inputs, but also understanding what inputs they have already and whether we can use them. 

I believe the two existing repos are at:
https://github.com/cmmid/covid-uk
and
https://github.com/mrc-ide/covid-sim

The later was fixed up by Microsoft and Github to have working CI and so on, and so it would be useful to know what testing they do, as we may be able to use something similar ourselves...

The reason we haven't used them before is that we've been explicitly told that RAMP don't want duplicates of their models, but we need a baseline to compare to, so it occurs to me that it's useful to onboard them anyway...

**Comment by alysbrett** [comment]
I have added @kzscisoft (Kristian) to this issue who started in our UKAEA RSE group today! His C++ skills and previous experience of working in particle large scientific collaborations and using DiRAC is likely to be useful. I also added @richardreeve as this is his idea and I think he intends to arrange for one or more epidemiologists to contribute to this work.

**Comment by richardreeve** [comment]
Yes, thanks, and welcome @kzscisoft! @LouiseMatthews is in charge of epidemiologists, so she will work this out officially, but I'm very happy to try to help anyway. In the meanwhile, we can clone the two repositories and make a team, and at least one label and project. I'll discuss it in the progress meeting tomorrow (Tues @ 3pm), so hopefully we'll pick up helpers there...

**Comment by richardreeve** [comment]
I've forked the two repositories because I'm not sure who has permissions to create new repos, and I've given the `Model-reconstruction` team admin rights over them. However, I noticed while doing so that https://github.com/mrc-ide/covid-sim is under heavy development. I think it's mostly to fix minor bugs and aesthetic issues from what I can see, but it may be worth starting with https://github.com/ScottishCovidResponse/covid-uk, whose parent is idle.

**Comment by kzscisoft** [comment]
Thanks for the welcome @richardreeve. After I have finished setting up my machine today I will fork the repositories myself too and see if I can get anything running as a preliminary trial keeping an eye on any environments I am using and perhaps trying to build within a container too so it can be reproduced.

**Comment by kzscisoft** [comment]
So I have started with the `covid-uk` repository which is written in R (I have not used the language before but I do not think that will be an issue). Couple of points when it comes to getting started, firstly the main script refers to a `Dropbox` location for the data when it should refer to the repository folder itself, secondly various R packages need to be installed. I think I will go ahead with the idea of getting this into a Docker container so it is easier for us all to reproduce results and run.

**Comment by kzscisoft** [comment]
@richardreeve I was wondering, should there be a chat stream for Model Reconstruction on Zulip or does this fall under the `existing-models` category? Thanks!
